.github/workflows/release.yaml

```diff
@@ -0,0 +1,28 @@
+name: release
+on:
+  push:
+    tags:
+      - "v*.*.*"
+  workflow_dispatch:
+jobs:
+  release-kubectl-plugin:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+      - name: GoReleaser
+        uses: goreleaser/goreleaser-action@v6
+        with:
+          distribution: 'goreleaser'
+          version: latest
+          args: release --clean
+          workdir: 'kubectl-plugin'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # TODO(MortalHappiness): This won't work now. The first version of plugin has to be submitted manually by use to the krew-index repo, so I'll submit a PR to the krew-index repo once a release candidate is created.
+      - name: Update new version in krew-index
+        uses: rajatjindal/krew-release-bot@v0.0.46
```

Comment:
Is this the standard project used with krew to publish plugin?

Reply:
It's mentioned in the official doc and says **strongly recommended**

https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/